env.example

```diff
@@ -4,6 +4,16 @@ NODE_ENV=development
 # PORT is the port used by the web server
 PORT=3000
```

Comment:
Does [this](https://github.com/Seneca-CDOT/telescope/pull/1649/files#diff-657de79094f16fa5feacfb48eb9d358b1060fb9b7d3256138b7f32b9c8c0fb22R8) change affect this?

Reply:
It does, thanks for bring it up. 
It seems like I need to make a conditional statement within the package.json
@humphd cc'ing just for clarity, Are we allowed to make conditionals in our packages for Telescope?

Reply:
I don't think they're related. `PORT=3000` is for telescope (telescope serving the back end), The change @chrispinkney is talking about ([this](https://github.com/Seneca-CDOT/telescope/pull/1649/files#diff-657de79094f16fa5feacfb48eb9d358b1060fb9b7d3256138b7f32b9c8c0fb22R8)) is for next's production server (a separate server to serve the production build, but telescope has nothing to do with it).